.github/workflows/update-renovate-pr.yml

```diff
@@ -0,0 +1,68 @@
+---
+# Renovate Pull Requests are not updating the license files in the licenses folder.
+# This workflow will checkout the renovate PR, update the license files, and push the changes back to the PR.
+name: update-renovate-pr
+
+on:
+  push:
+    branches:
+      - renovate/**
+    paths-ignore:
+      - licenses/**
+
+permissions:
+  contents: read
+
+jobs:
+  update-renovate-pr:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/workflows/gradle-goal
+        with:
+          command: "./gradlew updateLicensesAndNotice"
+
+      - name: Debug diff
+        run: git status --porcelain
+
+      - name: Check if licenses folder has changed
+        id: check-notice-diff
+        run: |
+          changed=false
+          if git status --porcelain | grep -q 'licenses/'; then
+            changed=true
+          fi
+          echo "changed=$changed" >> "$GITHUB_OUTPUT"
+
+      - name: Debug output
+        run: |-
+          echo 'changed: ${{ steps.check-notice-diff.outputs.changed }}'
+
+      ## this should help with of multiple files in a new folder.
+      - name: Add changes
```

Comment:
This is handy so we can run the push changes through all the files, otherwise all the files under a new folder won't be listed when using `git status --porcelain`